### PR TITLE
Init logger with log prefix.

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Initialize logger with log prefix

#128

### CODE REVIEW

File: logger/index.js [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
   transports: getWinstonTransports(),
   exitOnError: false
 });
 
 winston.addColors(getWinstonColors());
 
+const logger = function(prefix) {    
+  const prefixedLogger = {
+      error: function(text) {
+          winstonLogger.error(`[${prefix}] ` + text)
```
Comment (liayoo) [inline logger/index.js:15]: (minor) How about including the text in the string like this? \`[${prefix}] ${text}\`

File: logger/index.js [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+      },
+      info: function(text) {
+          winstonLogger.info(`[${prefix}] ` + text)
```
Comment (liayoo) [inline logger/index.js:18]: ditto

File: logger/index.js [hunk 3]
```diff
@@ -1,12 +1,28 @@
...
+      },
+      debug: function(text) {
+          winstonLogger.debug(`[${prefix}] ` + text)
```
Comment (liayoo) [inline logger/index.js:21]: ditto

**Review by liayoo — APPROVED** [review]
LGTM, with some minor comments. Thanks!

File: logger/index.js (hunk 1 above)
Comment (platfowner) [reply logger/index.js]: Done. Thanks!

File: logger/index.js (hunk 2 above)
Comment (platfowner) [reply logger/index.js]: ditto

File: logger/index.js (hunk 3 above)
Comment (platfowner) [reply logger/index.js]: ditto

**Comment by platfowner** [comment]
All comments addressed. Thanks. Now merging..

**Pull request merged**